internal/store/continuous_backup.go

```diff
@@ -37,14 +37,10 @@ type ContinuousJobCreator interface {
 	CreateContinuousRestoreJob(string, string, *atlas.ContinuousJobRequest) (*atlas.ContinuousJobs, error)
 }
 
-type SnapshotsLister interface {
+type ContinuousSnapshotsLister interface {
 	ContinuousSnapshots(string, string, *atlas.ListOptions) (*atlas.ContinuousSnapshots, error)
 }
 
-type SnapshotDescriber interface {
```

Comment:
[q] I am wondering why we are removing this. Do we have another API call for describing a snapshot?

Reply:
short answer, we don't have a command using it, if we need it we'll add it later